qtawesome/__init__.py

```diff
@@ -222,3 +223,26 @@ def set_defaults(**kwargs):
     """
     return set_global_defaults(**kwargs)
 
+
+class IconWidget(QtWidgets.QLabel):
+    '''
+    IconWidget gives the ability to display an icon as a widget
+    it supports setIcon() and setIconSize()
+    '''
+    def __init__(self, *names, **kwargs):
+        super().__init__()
```

Comment:
This is not the correct usage of `super` in Python 2.

Reply:
```suggestion
        super(IconWidget, self).__init__()
```